fixtures/showtec/performer-2500.json

```
@@ -0,0 +1,225 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Performer 2500",
+  "categories": ["Dimmer", "Color Changer"],
+  "meta": {
+    "authors": ["purple", "Luke Nelson"],
+    "createDate": "2022-10-18",
+    "lastModifyDate": "2022-10-18"
+  },
+  "links": {
+    "productPage": [
+      "https://www.highlite.com/en/33141-performer-2500-fresnel.html"
+    ],
+    "manual": [
+      "https://all-guidesbox.com/manual/2217476/showtec-performer-2500-fresnel-tungsten-operation-user-s-manual-40.html"
```

There is a better link directly on the manufacturer's product page. Linking to that one is preferred over those manual collecting websites.

```suggestion
      "https://www.highlite.com/en/mwdownloads/download/link/id/17429015/"
```